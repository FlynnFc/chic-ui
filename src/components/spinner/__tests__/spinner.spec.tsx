import Spinner from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Spinner Component', () => {
  it('Renders Correctly', () => {
    const SpinnerGroup = render(
      <>
        <Spinner size={40} />
        <Spinner type="primary" />
        <Spinner type="secondary" />
        <Spinner type="danger" />
        <Spinner type="success" />
        <Spinner type="warning" />
        <Spinner type="info" />
      </>
    );
    expect(SpinnerGroup).toMatchSnapshot();
  });
});
