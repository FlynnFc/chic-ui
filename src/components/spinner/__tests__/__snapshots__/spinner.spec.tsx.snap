// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Renders Correctly 1`] = `
Array [
  <span
    className="sc-bdfBQB hDjhaj"
    size={40}
  />,
  <span
    className="sc-bdfBQB bjKFrv"
    size={50}
  />,
  <span
    className="sc-bdfBQB eXZiUs"
    size={50}
  />,
  <span
    className="sc-bdfBQB bJSTre"
    size={50}
  />,
  <span
    className="sc-bdfBQB cWGJAL"
    size={50}
  />,
  <span
    className="sc-bdfBQB dJkocW"
    size={50}
  />,
  <span
    className="sc-bdfBQB biOdMA"
    size={50}
  />,
]
`;
